client/pages/consultations/[id]/results/index.vue

```diff
@@ -1,12 +1,26 @@
 <script setup lang="ts">
 
+import ConsultationAnswers from "~/client/types/consultation/consultationAnswers";
 definePageMeta({
   layout: 'basic'
 })
 
 const consultationId = useRoute().params.id.toString()
 const consultationResults = (await (new ConsultationApi().getConsultationResults(consultationId))).value
 
+const questionsPresentation: ConsultationAnswers[] = (
+  ['resultsOpen', 'resultsMultipleChoice','resultsUniqueChoice' ])
+  .flatMap((key) => ((consultationResults)[key])
+    .map(({ order, questionId, questionTitle, responses}) => ({
+      type: key,
+      order,
+      questionId,
+      questionTitle,
+      responses
+    })));
+
+questionsPresentation.sort((a,b) => a.order - b.order)
```

Comment:
Ça marche pas un truc de ce genre ?

```suggestion
const questions = [
  ...consultationResults.resultsOpen,
  ...consultationResults.resultsMultipleChoice,
  ...consultationResults.resultsUniqueChoice
].sort((a,b) => a.order - b.order)
```